### PR TITLE
Enable Reason of analyze schema issue in the callhome after obfuscating sensitive information

### PR DESCRIPTION
**Pull request by priyanshi-yb**
Enabling only Reason as I feel Suggestion is not important to send to callhome as it doesn't any valuable information about the user issues. Its the constant information which we already know.

### CODE REVIEW

File: yb-voyager/cmd/analyzeSchema.go [hunk 1]
```diff
@@ -1819,20 +1828,20 @@ func packAndSendAnalyzeSchemaPayload(status string) {
...
+				switch sensitiveReason {
+				case UNSUPPORTED_DATATYPE, UNSUPPORTED_DATATYPE_LIVE_MIGRATION:
+					//e.g. Reason "Unsupported datatype - xml on column - data"
+					//sending only "Unsupported datatype - xml"
+					issue.Reason = strings.Split(issue.Reason, "on column -")[0]
+				default:
+					issue.Reason = sensitiveReason
+				}
+			}
+		}
```
Comment (sanyamsinghal) [inline yb-voyager/cmd/analyzeSchema.go:1842]: basically you are checking and modifying the issueReason which have sensitive information?

**Review by sanyamsinghal — APPROVED** [review]
LGTM

File: yb-voyager/cmd/analyzeSchema.go [hunk 2]
```diff
@@ -1819,20 +1828,20 @@ func packAndSendAnalyzeSchemaPayload(status string) {
...
+		//I believe there is no point storing the suggestions as it is something we provide
+		//so we know what is the suggestion for a particular reason.
```
Comment (sanyamsinghal) [inline yb-voyager/cmd/analyzeSchema.go:1844]: nit:
```
// no need since we already have it in documentation
```

**Review by sanyamsinghal — APPROVED** [review]
LGTM

File: yb-voyager/cmd/analyzeSchema.go (hunk 1 above)
Comment (priyanshi-yb) [reply yb-voyager/cmd/analyzeSchema.go:1842]: right

**Pull request merged**